synapse/cortex.py

```diff
@@ -121,6 +121,29 @@
     'required': [],
 })
 
+reqValidStormMacro = s_config.getJsValidator({
+    'type': 'object',
+    'properties': {
+        'name': {'type': 'string', 'pattern': '^.{1,491}$'},
+        'iden': {'type': 'string', 'pattern': s_config.re_iden},
+        'user': {'type': 'string', 'pattern': s_config.re_iden},
```

Comment:
should this be `creator` rather than `user`?

Reply:
It should, but i was trying to stay backward compatible